lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -37,8 +40,8 @@ def store_column(column_title, format_string, new_data):
     store_column("time (pixels)", "%.18e", times_idx)
     store_column("coordinate (pixels)", "%.18e", coords_idx)
 
-    store_column("time", "%.18e", seconds)
-    store_column("position", "%.18e", position)
+    store_column(f"time ({time_units})", "%.18e", seconds)
+    store_column(f"position ({position_units})", "%.18e", position)
```

Comment:
Changing this is a breaking change (data format change). Should be in the changelog.

I totally support the change though. We should always put units there.